tests/core/a-animation.test.js

```diff
@@ -118,6 +118,34 @@ suite('a-animation', function () {
     });
   });
 
+  suite('component color animation', function () {
```

Comment:
Would be nice to have tests for:

- Shorthand `#fff`
- Capital `#FFF`
- Interpolated value. Check that between the animation, the color is somewhere between the start and end.